mainfile

```diff
@@ -10,35 +10,35 @@
     DC MOTORS AND AN ULTRASONIC SENSOR. 
 */
 
```

Comment:
**Time testing**
1. for(int i = 0; i < 20; i++){
         analogWrite(motor1,2,3,4, normal);
    }
2. Set up a distance measurement method (line marking 4 ft, etc.)
3. Have the car move forward (code above) until it reaches the 4 ft marker and time how long it takes to do so
4. Divide that time by 4 and that is our time per foot
5. for(int i = 0; i < 20; i++){
         analogWrite(motor1,3, normal);
         analogWrite(motor2,4, slow);
    }
6. Have the car drive in a circle (360 degrees) (code above)
7. Time how long it takes to drive 360 degrees
8. Divide that time by 360 and that is our time per degree
9. Use time per foot and time per degree in our moveForward and turning methods